packages/app/components/tabs/constants.ts

```diff
@@ -0,0 +1,8 @@
+import { twMerge } from "tailwind-merge";
+
+import { buttonStyles } from "@/ui";
+
+export const tabButtonStyles = twMerge(
+  buttonStyles({ variant: "secondary" }),
+  "bg-transperant border border-transparent text-em-high font-semibold ui-selected:bg-surface-75 ui-not-selected:text-em-low  hover:bg-surface-50 hover:border-surface-75 focus:ring-0"
+);
```

Comment:
this doesn't make sense to be a constant. It's a style.

Renaming it for styles.ts would be an improvement.

Reply:
Done!